support-frontend/assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -47,6 +47,21 @@ body {
   background-color: gu-colour(sport-faded);
 }
 
+.gu-content__main.aus-moment {
+  position: relative;
+  background: linear-gradient(
+      #FFE500 50%,
+      #F3C100 50%,
+      #F3C100 80%,
+      #FF7F0F 80%
+  ) !important;
+}
+
+// This selects div.gu-content__blurb and overrides the default sticky positioning
```

Comment:
why can't we target `gu-content__blurb` instead of using `first-child`?

Reply:
We totally can @tomrf1, thanks for pointing that out! I meant to change it before pushing 🙄